addons/revolutionaries/CfgGroups_Indep.hpp

```diff
@@ -0,0 +1,236 @@
+                position[] = {0, 0, 0};
+            };
+            class unit1 {
+                vehicle = "TACU_Revolutionaries_U_I_Russian_Rifleman04";
+                side = 2;
+                rank = "PRIVATE";
+                position[] = {5, -5, 0};
+            };
+        };		
+		
+        class TACU_Revolutionaries_G_I_Russian_FTeamL {
+            name = "Fireteam";
+            side = 2;
+            faction = "TACU_Revolutionaries_O";
+            icon = "\a3\ui_f\data\map\markers\nato\o_inf.paa";
+            class unit0 {
+                vehicle = "TACU_Revolutionaries_U_I_Russian_Rifleman01";
+                side = 2;
+                rank = "SERGEANT";
+                position[] = {0, 0, 0};
+            };
+            class unit1 {
+                vehicle = "TACU_Revolutionaries_U_I_Russian_Rifleman02";
+                side = 2;
+                rank = "PRIVATE";
+                position[] = {5, -5, 0};
+            };
+            class unit2 {
+                vehicle = "TACU_Revolutionaries_U_I_Russian_Rifleman04";
+                side = 2;
+                rank = "PRIVATE";
+                position[] = {-5, -5, 0};
+            };
+            class unit3 {
+                vehicle = "TACU_Revolutionaries_U_I_Russian_Base";
+                side = 2;
+                rank = "PRIVATE";
+                position[] = {10, -5, 0};
+            };
+        };
+		
+        class TACU_Revolutionaries_G_I_Russian_SquadL {
+            name = "Squad";
+            side = 2;
+            faction = "TACU_Revolutionaries_O";
+            icon = "\a3\ui_f\data\map\markers\nato\o_inf.paa";
+            class unit0 {
+                vehicle = "TACU_Revolutionaries_U_I_Russian_Rifleman01";
+                side = 2;
+                rank = "SERGEANT";
+                position[] = {0, 0, 0};
+            };
+            class unit1 {
+                vehicle = "TACU_Revolutionaries_U_I_Russian_Rifleman02";
+                side = 2;
+                rank = "PRIVATE";
+                position[] = {5, -5, 0};
+            };
+            class unit2 {
+                vehicle = "TACU_Revolutionaries_U_I_Russian_Rifleman04";
+                side = 2;
+                rank = "PRIVATE";
+                position[] = {-5, -5, 0};
+            };
+            class unit3 {
+                vehicle = "TACU_Revolutionaries_U_I_Russian_Base";
+                side = 2;
+                rank = "PRIVATE";
+                position[] = {10, -5, 0};
+            };
+            class unit4 {
+                vehicle = "TACU_Revolutionaries_U_I_Russian_Rifleman02";
+                side = 2;
+                rank = "PRIVATE";
+                position[] = {-10, -5, 0};
+            };
+            class unit5 {
+                vehicle = "TACU_Revolutionaries_U_I_Russian_Rifleman04";
+                side = 2;
+                rank = "PRIVATE";
+                position[] = {15, -5, 0};
+            };
+        };
+	 };
```

Comment:
Formatting on this one and spaces instead of tabs.
```suggestion
    };
```